mappings/net/minecraft/block/Material.mapping

```diff
@@ -4,45 +4,45 @@ CLASS net/minecraft/class_3614 net/minecraft/block/Material
 	FIELD field_15913 COBWEB Lnet/minecraft/class_3614;
 	FIELD field_15914 STONE Lnet/minecraft/class_3614;
 	FIELD field_15915 BUBBLE_COLUMN Lnet/minecraft/class_3614;
-	FIELD field_15916 SAND Lnet/minecraft/class_3614;
+	FIELD field_15916 AGGREGATE Lnet/minecraft/class_3614;
 	FIELD field_15917 SPONGE Lnet/minecraft/class_3614;
 	FIELD field_15918 REDSTONE_LAMP Lnet/minecraft/class_3614;
 	FIELD field_15919 PORTAL Lnet/minecraft/class_3614;
 	FIELD field_15920 WATER Lnet/minecraft/class_3614;
 	FIELD field_15921 UNUSED_PLANT Lnet/minecraft/class_3614;
 	FIELD field_15922 LAVA Lnet/minecraft/class_3614;
 	FIELD field_15923 LEAVES Lnet/minecraft/class_3614;
-	FIELD field_15924 PART Lnet/minecraft/class_3614;
+	FIELD field_15924 DECORATION Lnet/minecraft/class_3614;
 	FIELD field_15925 CACTUS Lnet/minecraft/class_3614;
-	FIELD field_15926 SEAGRASS Lnet/minecraft/class_3614;
+	FIELD field_15926 REPLACEABLE_AQUATIC_PLANT Lnet/minecraft/class_3614;
 	FIELD field_15927 STRUCTURE_VOID Lnet/minecraft/class_3614;
-	FIELD field_15928 PACKED_ICE Lnet/minecraft/class_3614;
+	FIELD field_15928 DENSE_ICE Lnet/minecraft/class_3614;
 	FIELD field_15929 liquid Z
 	FIELD field_15930 EGG Lnet/minecraft/class_3614;
 	FIELD field_15931 WOOL Lnet/minecraft/class_3614;
 	FIELD field_15932 WOOD Lnet/minecraft/class_3614;
 	FIELD field_15933 PISTON Lnet/minecraft/class_3614;
 	FIELD field_15934 SNOW_BLOCK Lnet/minecraft/class_3614;
 	FIELD field_15935 PLANT Lnet/minecraft/class_3614;
-	FIELD field_15936 CLAY Lnet/minecraft/class_3614;
+	FIELD field_15936 MOB_DERIVED Lnet/minecraft/class_3614;
 	FIELD field_15937 CAKE Lnet/minecraft/class_3614;
-	FIELD field_15938 BAMBOO_SAPLING Lnet/minecraft/class_3614;
+	FIELD field_15938 BAMBOO_SHOOT Lnet/minecraft/class_3614;
 	FIELD field_15939 blocksLight Z
 	FIELD field_15940 blocksMovement Z
-	FIELD field_15941 EARTH Lnet/minecraft/class_3614;
+	FIELD field_15941 TOPSOIL Lnet/minecraft/class_3614;
 	FIELD field_15942 GLASS Lnet/minecraft/class_3614;
 	FIELD field_15943 FIRE Lnet/minecraft/class_3614;
 	FIELD field_15944 color Lnet/minecraft/class_3620;
-	FIELD field_15945 ORGANIC Lnet/minecraft/class_3614;
+	FIELD field_15945 FODDER Lnet/minecraft/class_3614;
 	FIELD field_15946 BAMBOO Lnet/minecraft/class_3614;
-	FIELD field_15947 UNDERWATER_PLANT Lnet/minecraft/class_3614;
-	FIELD field_15948 SNOW Lnet/minecraft/class_3614;
-	FIELD field_15949 ANVIL Lnet/minecraft/class_3614;
+	FIELD field_15947 AQUATIC_PLANT Lnet/minecraft/class_3614;
+	FIELD field_15948 SNOW_LAYER Lnet/minecraft/class_3614;
```

Comment:
Whats the reasoning behind this? SNOW seems fine?

Reply:
I agree, especially since the block is `minecraft:snow`.

Reply:
To make it clearer between SNOW_BLOCK and SNOW

Reply:
I also want to clarify this since 1.17 (iirc the right version) will have "snowier snow," so I want to clarify this as much as possible so that it doesn't become out of date soon

Reply:
We don't yet know what Mojang is going to do. I'd rather change it later than change it now and later find out the change was in the wrong direction.

Reply:
As it stands now, this name still clarifies the blocks that use this material.